crocodealer.cabal

```diff
@@ -25,11 +25,16 @@ library
   exposed-modules:     Crocodealer
                          Crocodealer.Config
                          Crocodealer.Core.Label
+                         Crocodealer.Command.Sync
+                         Crocodealer.Command.Colors
```

Comment:
Since coloring is not a command, it's better to move this module outside, like `Crocodealer.ColorTerminal`